src/EPR.Calculator.API.UnitTests/CalcResultsExporterTests.cs

```diff
@@ -130,10 +131,16 @@ public void Export_CreatesCorrectCsvContent()
                 }
             };
 
+            var expectedFileName = $"{calcResult.CalcResultDetail.RunId}" +
+                $"-{calcResult.CalcResultDetail.RunName}" +
+                $"_Results File" +
+                $"_{DateTime.Now:yyyyMMdd}.csv";
```

Comment:
This code might not be run on the Dev11 as it uses Non-BST time. Can we do anything about it? Thanks